app/src/engineering/java/com/hedvig/app/feature/embark/EmbarkMockActivity.kt

```diff
@@ -49,6 +50,11 @@ class EmbarkMockActivity : MockActivity() {
     })
 
     override fun adapter() = genericDevelopmentAdapter {
+        header("Previous Insurer")
+        clickableItem("Previous Insurer") {
+            MockEmbarkViewModel.mockedData = PREVIOUS_INSURER_STORY
+            startActivity(EmbarkActivity.newInstance(this@EmbarkMockActivity, this.javaClass.name))
```

Comment:
I got tired of writing `this@WhateverMockActivityIWasIn`, so I created a

```kotlin
val context: Context
	get() = this
```

in `MockActivity`, which enables the following:

```suggestion
            startActivity(EmbarkActivity.newInstance(context, this.javaClass.name))
```